testkit/src/main/java/org/apache/calcite/test/SqlOperatorTest.java

```diff
@@ -6224,6 +6224,30 @@ private void testCurrentDateFunc(Pair<String, Hook.Closeable> pair) {
     f.checkNull("STRPOS(x'', null)");
   }
 
+  @Test void testInstrFunction() {
+    final SqlOperatorFixture f0 = fixture().setFor(SqlLibraryOperators.INSTR);
+
```

Comment:
you should add a test that ensures the function doesnt work unless its set for bigquery, you should also test against each library the function supports, not just bigquery. good example [here](https://github.com/apache/calcite/blob/e5f7729d7d44630fc9c3b06bc326f335089b4b84/testkit/src/main/java/org/apache/calcite/test/SqlOperatorTest.java#L1756).